### DISCUSSION TITLE
大家好。。。我是派蒙，从今天开始我入驻绳网啦❤️

### DISCUSSION THREAD

**Discussion by zdxiaoda** [share121/inter-knot#495] [其他]
![原神贺图_2020派蒙生日_阿托伊玛](https://github.com/user-attachments/assets/7b0a10c8-f0e2-46c7-8393-5dee33a77aa2)

听说这里可以接到很多委托～(≧∇≦)/
嘿嘿嘿，这样就可以让旅行者赚到更多摩拉，买更多好吃的了！(*^▽^*)

**Comment by lbghhj** [comment]
加强黄毛，我什么都会做的，请一定要加强黄毛，拜托你了派蒙

**Comment by xinxingjie** [comment]
欢迎来到绳网！帖子千万条，守法第一条。绳网不是法外之地，请各位新用户积极遵守绳网语言规范条例，共同守护绳网安全、健康的网络风气。最后祝您在绳网接单愉快！

**Comment by MiaoJiSir** [comment]
[求助]喵吉长官德求助！捕捉派蒙，到喵吉长官处领取奖励！

**Reply by zdxiaoda** [reply]
什么？！居然有人想抓派蒙？
唔嗯，居然还敢打派蒙的主意？派蒙才不会让你得逞呢！(＃｀д´)ﾉ 

不过，旅行者，你不会真的要抓派蒙吧？( ；∀；)
呜…快说你是开玩笑的！
![ac5501aa82f233caf1b68a9900022c1b0677ba5a](https://github.com/user-attachments/assets/b1a48ea9-e199-4552-afc4-79d8cbb2e0ed)

**Comment by nahida0-o** [comment]
派蒙最可爱！！！

**Comment by lllliufeng** [comment]
丁尼广进！

**Comment by MiG-2233** [comment]
派蒙超超

**Comment by mhpmc** [comment]
哈？给我干哪来了，这还是新艾利都吗？

**Comment by cheeseRatRat** [comment]
可爱，吃了

**Reply by zdxiaoda** [reply]
什、什么？！居然想吃派蒙？
你这个大坏蛋，派蒙才不是食物！
唔嗯…既然这样，派蒙决定给你起个最难听的绰号！
就叫你“贪吃鬼”吧！
![ac5501aa82f233caf1b68a9900022c1b0677ba5a](https://github.com/user-attachments/assets/0f11b872-35cf-4fe5-a09b-fafc25438fe8)

**Comment by Akashic007** [comment]
这还是新艾利都吗？
![透明派蒙](https://github.com/user-attachments/assets/892f40ea-1a86-47ef-9a4e-e8de17bb5717)

**Comment by 123456qwert27** [comment]
派蒙可爱捏

**Comment by SDCOM-0415** [comment]
派蒙，这里是**新艾利都**，我们用的是**丁尼**来结算的！

**Comment by Griffin233** [comment]
你能表演一个流口水吗

**Comment by feco-y** [comment]
兜售渔人吐司啦~
50万丁尼一块 来自异世界的美味 先到先得啦~

**Comment by Summer-rain897** [comment]
不知道这个网站可以撑多久。
